theme_kea/static/src/scss/primary_variables.scss

```diff
@@ -1,77 +1,20 @@
-//------------------------------------------------------------------------------
```

Comment:
Keep this part for compatibility purposes.

Reply:
@xlu-odoo You still not handled that before asking for review today? :thinking: